packages/suite/src/types/suite/github.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable camelcase */
+
+export type ReleaseInfo = {
```

Comment:
Could be included from a dependency but I would prefer not to include yet another package in the project where we'll use not even 1% of it. These types are based on the GH API documentation.

Reply:
Please add a comment with link to those docs if possible. Thanks

Reply:
> Could be included from a dependency but I would prefer not to include yet another package in the project where we'll use not even 1% of it. These types are based on the GH API documentation.

I see. I guess we're concerned about polluting the release rather than the repository/project itself, right? Isn't there any way to 'cherry pick' just a subset of a dependency that would be included in the release?

Reply:
esbuild supports tree shaking, so yes it is indeed possible to reduce the amount of code making into a release. This is not really a concern here as the whole library would likely add very little weight (few KB) to the desktop bundle that is already +100MB (most of it is electron).

It is important to try to keep the amount of libraries to the minimum necessary. The bloating of the `node_modules` folder is an obvious reason, but most importantly, it's about resiliency and security.

In the NPM ecosystem, packages rely a lot on other packages, which themselves rely on more, and so on. If anything that gets pulled or compromised in the chain, by having a lot of dependencies, we increase the risk of this happening to us. Previous incidents: https://en.wikipedia.org/wiki/Npm_(software)#Notable_breakages

Reply:
I agree with @keraf on this. It is indeed more than fine to simple copy & paste a bit of code than adding yet another NPM dependency.

Reply:
Allright. Thanks for exhaustive answser.